components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md

```diff
@@ -77,6 +77,19 @@ You can mix and match different services to add nodes across various services.
     ```sh
     chef-automate node add --automate-ips 10.1.2.23 --chef-server-ips 10.1.0.36,10.0.1.233  --postgresql-ips 10.0.1.42,10.0.1.54
     ```
+You can remove unreachable nodes while adding nodes
+
+- To remove unreachable nodes while adding new nodes, pass the flag `--remove-unreachable-node` or `-r` like example below
```

Comment:
Replace it with: - To remove unreachable nodes while adding new nodes, pass the flag `--remove-unreachable-node` or `-r` like the example below: